Sample/Vungle Sample App/BannerList.m

```diff
@@ -0,0 +1,139 @@
+//
+//  BannerList.m
+//  Vungle Sample App
+//
+//  Created by John Mai on 2/6/20.
+//  Copyright © 2020 Jordyn Chuhaloff. All rights reserved.
+//
+
+#import "BannerList.h"
+#import "Constants.h"
+#import "DisplayTableViewCell.h"
+
+@interface BannerList () {
+    CGFloat screenHeight;
+    CGFloat screenWidth;
+}
+@property (strong, nonatomic) IBOutlet UIView *view;
+@property const int totalSize;
+@property const int AD_PER_ITEM;
```

Comment:
This var declaration doesn't fit the format used for the rest of the variables in the class (this is more close to the type of syntax used for constants). I'd recommend changing this to the expected camel-case format (i.e., `adPerItem`).